### PR TITLE
Add File Does not Exists Test

### PR DESCRIPTION
**Pull request by ArlonAntonius**
Simple test, was not sure on where to place it.

### CODE REVIEW

File: tests/unit/Tenancy/Database/DatabaseResolverTest.php [hunk 1]
```diff
@@ -58,4 +60,20 @@ public function resolves_driver()
         $this->assertEquals(Tenancy::getTenantConnectionName(), $connection->getConfig('name'));
         $this->assertEquals($this->tenant->getTenantKey(), $connection->getConfig('tenant-key'));
     }
+
+
+
```
Comment (luceos) [inline tests/unit/Tenancy/Database/DatabaseResolverTest.php]: Too many newlines here.
```suggestion
```

File: tests/unit/Tenancy/Database/Mocks/DatabaseDriver.php [hunk 2]
```diff
@@ -26,11 +27,15 @@ class DatabaseDriver implements ProvidesDatabase
      */
     public function configure(Tenant $tenant): array
     {
-        return [
```
Comment (luceos) [inline tests/unit/Tenancy/Database/Mocks/DatabaseDriver.php:29]: 😱 😱

**Review by luceos — CHANGES_REQUESTED** [review]
One suggestion. Thanks.

File: tests/unit/Tenancy/Database/Mocks/DatabaseDriver.php (hunk 2 above)
Comment (ArlonAntonius) [reply tests/unit/Tenancy/Database/Mocks/DatabaseDriver.php:29]: What? You kinda forgot about this?

**Pull request merged**